### DISCUSSION TITLE
antd tabs and tabpane

### DISCUSSION THREAD

**Discussion by youcefTab** [refinedev/refine#2058] [Q&A]
Hello team Refine. 

i'm using in my project the tabs from antd, there is an example: https://codesandbox.io/s/f34pyu?file=/demo.js

i'm using basicly the same code structure as the example, the difference is that i put it inside a <List> : 
```
return <div>
  {some other divs}
  <List> ... </List>
</div>. 
```
The Problem that i have is that when i add multiple tabs, i don't get the three dots, the tabs just overflow. i tried multiple things using custom css but i don't get it to work. can i get any help on this ?

**Comment by omeraplak** [comment]
Hey @youcefTab ,
hmm, looks like this is a CSS issue. @salihozdemir , @aliemir , do you have any ideas?

**Reply by youcefTab** [reply]
Hey @omeraplak , i noticed something else, in <List> Component i tried to add 3 buttons in pageHeaderProps:{{extra}}, when i reduce the size of the page the buttons go outside the Content layout, so i guess there is a CSS problem in the Content layout
